labs.yml

```diff
@@ -43,3 +43,10 @@ commands:
 
   - name: create-table-mapping
     description: create initial table mapping for review
+
+  - name: ensure-assessment-run
+    description: ensure the assessment job was run on a workspace
+    flags:
+      - name: run
+        description: run the assessment workflow if it hasn't been successfully run
```

Comment:
let's run it by default if it's not there, no need for an option

Reply:
Done